docs/content/preview/yugabyte-cloud/cloud-faq.md

```diff
@@ -39,7 +39,7 @@ For information on YugabyteDB Managed pricing, refer to the [YugabyteDB Managed
 
 Refer to [Cloud provider regions](../release-notes/#cloud-provider-regions) for a list currently supported regions.
 
-YugabyteDB Managed supports all the regions that have robust infrastructure and sufficient demand from customers. We are continuously improving region coverage, so if there are any regions you would like us to support, reach out to {{<support-cloud>}}.
+YugabyteDB Managed supports all the regions that have robust infrastructure and sufficient demand from customers. We are continuously improving region coverage, so if there are any regions you would like us to support, reach out to {{%support-cloud%}}.
```

Comment:
Suggestion: 
We are continuously improving region coverage, so if there are any regions you would like us to support, ... 
>
There is continuous expansion of region coverage, so if you require support for additional regions, ...